### ISSUE TITLE
CNN-Based Classifier

### ISSUE THREAD

**Issue by Hasanmog** [Hasanmog/CNN-VS-ViT#1]
- EuroSAT dataset
CNN-Based:
    - 89% test accuracy
- UC-Merced
   - 87% accuracy on test set
- [x] train
- [x] Apply Gradio
- [x] PEFT / Fine-tune / train on a higher quality dataset
